test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresUnreferencedCodeCapability.cs

```diff
@@ -651,15 +652,43 @@ class GenericWithStaticMethod<T>
 				public static void GenericTypeWithStaticMethodWhichRequires () { }
 			}
 
+			class GenericWithConstraint<T> where T : new()
+			{
+				public static void Generic<T> (T toDisplay) { }
+			}
+
+			class ClassWithRequiresUnreferencedCodeOnConstructor
+			{
+				[RequiresUnreferencedCode ("Message for --ClassWithRequiresUnreferencedCodeOnCostructor--")]
+				void ClassWithRequiresUnreferencedCodeOnCostructor () { }
+			}
+
 			[ExpectedWarning ("IL2026", "--GenericTypeWithStaticMethodWhichRequires--")]
 			public static void GenericTypeWithStaticMethodViaLdftn ()
 			{
 				var _ = new Action (GenericWithStaticMethod<TestType>.GenericTypeWithStaticMethodWhichRequires);
 			}
 
+			public static void GenericCallUsingMakeGenericType ()
```

Comment:
Nit: I would add the `[ExpectedWarning]` and comment it out - with a reference to the issue. Just that it's obvious what's wrong and what should the "fix" handle.